deploy/example_catalog/cr-app-cdh632cm.json

```diff
@@ -0,0 +1,415 @@
+                "label": {
+                    "name": "HMaster"
+                }
+            },
+            {
+                "endpoint": {
+                    "isDashboard": true,
+                    "urlScheme": "http",
+                    "port": 16030,
+                    "path": "/"
+                },
+                "id": "hbase-worker",
+                "label": {
+                    "name": "HRegionServer"
+                }
+            },
+            {
+                "endpoint": {
+                    "isDashboard": false,
+                    "port": 20550
+                },
+                "id": "hbase-rest",
+                "label": {
+                    "name": "HBase REST"
+                }
+            },
+            {
+                "endpoint": {
+                    "isDashboard": true,
+                    "urlScheme": "http",
+                    "port": 9095,
+                    "path": "/"
+                },
+                "id": "hbase-thrift",
+                "label": {
+                    "name": "HBase Thrift Server"
+                }
+            },
+            {
+                "id": "hive-metastore",
+                "label": {
+                    "name": "HIVE Metastore"
+                }
+            },
+            {
+                "endpoint": {
+                    "isDashboard": true,
+                    "urlScheme": "http",
+                    "port": 10002,
+                    "path": "/"
+                },
+                "id": "hive-server",
+                "label": {
+                    "name": "HIVE Server 2"
+                }
+            },
+            {
+                "endpoint": {
+                    "isDashboard": false,
+                    "port": 50111
+                },
+                "id": "webhcat",
+                "label": {
+                    "name": "WebHCat Server"
+                }
+            },
+            {
+                "endpoint": {
+                    "isDashboard": true,
+                    "urlScheme": "http",
+                    "port": 8888,
+                    "path": "/"
+                },
+                "id": "hue",
+                "label": {
+                    "name": "Hue Console"
+                }
+            },
+            {
+                "endpoint": {
+                    "isDashboard": true,
+                    "urlScheme": "http",
+                    "port": 18088,
+                    "path": "/"
+                },
+                "id": "spark-history",
+                "label": {
+                    "name": "Spark history server"
+                }
+            },
+            {
+                "id": "impala",
+                "label": {
+                    "name": "Impala service"
+                }
+            },
+            {
+                "id": "oozie",
+                "label": {
+                    "name": "OOZIE service"
+                }
+            }
```

Comment:
Several of these services do not have endpoints. We can remove it?

Reply:
@swamibluedata We are declaring all the services with or without end points in the "services" key. The id of all the services is being also referred in "serviceIDs" key within "roleServices" key where we specify which service is running on which role. So we will not be able to remove these services.